dspace-spring-rest/src/test/java/org/dspace/app/opensearch/OpenSearchControllerTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.opensearch;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.xpath;
+
+import org.dspace.app.rest.test.AbstractControllerIntegrationTest;
+
+import org.dspace.services.ConfigurationService;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Integration test to test the /opensearch endpoint
+ * (Class has to start or end with IT to be picked up by the failsafe plugin)
+ *
+ * @author Oliver Goldschmidt (o dot goldschmidt at tuhh dot de)
+ */
+public class OpenSearchControllerTest extends AbstractControllerIntegrationTest {
+
+    // configuration is taken from dspace-api/src/test/data/dspaceFolder/config/local.cfg
+    private ConfigurationService configurationService;
+
+    @Before
+    public void init() throws Exception {
+        // override the configuration settings here if other settings are required for test
+        // configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+        // configurationService.setProperty("websvc.opensearch.enable", true);
+    }
```

Comment:
@olli-gold : Is this `init()` method even needed?  Looks like the code is all commented out here.  I suspect this can be removed, since you've enabled opensearch via the Test Framework's local.cfg file.

Reply:
You are absolutely right, the method is unnecessary and I have removed it with the next commit. Thank you for reviewing!